src/go/rpk/pkg/cli/cmd/cluster/metadata.go

```diff
@@ -162,99 +162,165 @@ func printBrokers(controllerID int32, brokers kadm.BrokerDetails) {
 	}
 }
 
-func PrintTopics(topics kadm.TopicDetails, internal, detailed bool) {
-	if !detailed {
+// Used for structured output. Note: structured output is always detailed and doens't require the -d flag.
```

Comment:
doens't => doesn't,
also can this be wrapped to 80 col?

Reply:
I've installed a vscode spell check plugin, sorry about the consistent silly typos here but that will stop going forward. As noted in another comment, these long comments are probably not needed anyway  so I'll remove the offending long comment which fixes the typos as well.